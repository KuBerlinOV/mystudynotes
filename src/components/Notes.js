import React from 'react';
import NotesList from './NotesList'
import NotesListFilters from './NotesListFilters';
import NoteForm from './NoteForm';



const Notes = ({ info }) => {


    return (
        <div>
            <section>

                <NotesListFilters />
                <h1>My notes: </h1>
                <NoteForm />
                <NotesList />
            </section>
        </div>
    )
}

export default Notes