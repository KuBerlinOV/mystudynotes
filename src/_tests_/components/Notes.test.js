import React from 'react';
import { shallow } from 'enzyme';
import Notes from '../../components/Notes';

describe('Notes', () => {
    it('should render Notes page', () => {
        const wrapper = shallow(<Notes />)
        expect(wrapper).toMatchSnapshot();
    })
})