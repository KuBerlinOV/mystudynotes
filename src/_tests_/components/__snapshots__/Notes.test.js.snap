// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notes should render Notes page 1`] = `
<div>
  <section>
    <Connect(NotesListFilters) />
    <h1>
      My notes: 
    </h1>
    <button>
      <Link
        to="/addnote"
      >
        Create Note
      </Link>
    </button>
    <Connect(NotesList) />
  </section>
</div>
`;
